test/linalg.jl

```diff
@@ -35,11 +35,11 @@ end
     rng = Random.MersenneTwister(1234321)
     df = (Y = randn(rng, 400), A = repeat(['N','Y'], outer=200),
         G = repeat('A':'T', inner = 2, outer=10), H = repeat('a':'j', inner=40))
-    lmm1 = fit!(LinearMixedModel(@formula(Y ~ 1+A+(1+A|G)+(1+A|H)), df,
-        wts  = ones(400)))
-    @test loglikelihood(lmm1) ≈ -578.9080978272708
-    MixedModels.reweight!(lmm1, ones(400))
-    @test loglikelihood(fit!(lmm1)) ≈ -578.9080978272708
+    m1 = fit(MixedModel, @formula(Y ~ 1 + A + (1+A|G) + (1+A|H)), df)
+    wm1 = fit(MixedModel, @formula(Y ~ 1+A+(1+A|G)+(1+A|H)), df, wts  = ones(400))
+    @test loglikelihood(wm1) ≈ loglikelihood(m1)
```

Comment:
Could keep the hardcoded value, doesn't hurt.

Reply:
For this, the most important thing isn't the unweighted fit, but rather that the unity-weighted fit has the same LL as the unweighted one.

There's a bug in re-weighting anyway (#334) that I may backport as well if I don't get the show-stoppers for 3.0 done in the next week.